data/json/items/ammo/9x18.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id" : "9x18mm",
```

Comment:
Style. Everything else in this folder uses the syntax without the leading space before the colon